doc/FAQ.rst

```diff
@@ -59,49 +65,131 @@ Conversely, what Panel adds on top of Bokeh is full bidirectional communication
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports R ggplot plots               | Yes             | No                   | No              | No                 | No                     | Yes                |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Altair/Vega plots            | Yes             | Yes                  | No              | Yes                | With a separate adapter| Yes                |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Django and Django channels   | Yes             | No                   | Yes             | No                 | No?                    | No?                |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Allows separating content from        | Yes             | No, but could with   | No              | No                 | No                     | No                 |
+|presentation                          |                 | traitlets            |                 |                    |                        |                    |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Servers supported                     | Jupyter, Bokeh, | Yes                  | Bokeh, Voila    | Streamlit          | Dash                   | Shiny server       |
+|                                      | Voila           |                      |                 |                    |                        |                    |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+
+Each of these libraries are free, open-source software packages, but all of them can be used with the commercial 
+`Anaconda Enterprise (AE5) <https://www.anaconda.com/enterprise/>`__ server product, and some can be used with other commercial servers 
+(Shiny, with `Shiny Server <https://www.rstudio.com/products/shiny-server-pro>`__, Streamlit, with `Streamlit Teams`, and Dash, with
+`Plotly Enterprise <https://plot.ly/products/on-premise>`__), to provide on-premises authenticated deployment within a private network.  Most of the servers (including Jupyter, Bokeh Server, Voila, and Dash) can be deployed on the public sites `mybinder.org <https://mybinder.org>`__ or `heroku <https://www.heroku.com>`__.
```

Comment:
from "Plotly Enterprise" to "Dash Enterprise" at https://plot.ly/dash/

Reply:
Thanks for the fix!